components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -102,7 +102,10 @@ module aero_model
   real(r8)          :: sol_factic_interstitial = 0.4_r8
   real(r8)          :: seasalt_emis_scale
   real(r8)          :: small = 1.e-36
-
+!<shanyp 01132025
```

Comment:
since git will keep track of when and who added these lines, all comments relating to author and date of changes should be removed.